src/main/kotlin/moody/blackjack/domain/card/Card.kt

```diff
@@ -0,0 +1,23 @@
+package moody.blackjack.domain.card
+
+class Card private constructor(suit: String, denomination: String) {
+    val suit: Suit
+    val denomination: Denomination
+
+    init {
+        this.suit = Suit.from(suit)
+        this.denomination = Denomination.from(denomination)
+    }
+
+    fun getScore(): Score {
+        return denomination.score
+    }
+
+    fun isAce() = denomination.isAce()
+
+    companion object {
+        fun of(suit: String, denomination: String): Card {
+            return Card(suit, denomination)
+        }
+    }
```

Comment:
```suggestion
    companion object {
        fun of(suit: Suit, denomination: Denomination): Card {
            return Card(suit.suit, denomination.denomination)
        }
    }
```
현재 테스트 코드에서 문자열로 카드의 suit, denomination 값을 입력받고 있는데요. 문자열 철자가 틀렸을 경우 알아차리기 어려울 수 있을 것 같아요!  Suit 과 Denomination 을 인자로 받는 건 어떻게 생각하시나요?

Reply:
외부에서 알 필요가 없는 객체는 내부에서만 관리하는 게 좋다고 생각해서 이런 방식으로 정했습니다! Card가 생성될 때 외부에서 Suit와 Denomination을 알 필요 없이 내부 생성자에서 각각의 원시값 포장 클래스의 .from() 메서드를 통해 검증을 포함한 객체 생성이 이루어집니다.